#ifndef __HGuideAPI_uart_baud_type_t_h__
#define __HGuideAPI_uart_baud_type_t_h__
#pragma once

// Custom Uart Baudrates
enum uart_baud_type_t
{
	uart_baud_4800       = 3,
	uart_baud_9600       = 4,
	uart_baud_19200      = 5,
	uart_baud_38400      = 6,
	uart_baud_57600      = 7,
	uart_baud_115200     = 8,
	uart_baud_230400     = 9,
	uart_baud_307200     = 10,
	uart_baud_460800     = 11,
	uart_baud_921600     = 12,
	uart_baud_1000000    = 13,
	uart_baud_2000000    = 14,
	uart_baud_3000000    = 15,
	uart_baud_4000000    = 16
};

#endif // __HGuideAPI_uart_baud_type_t_h__
